src/components/Navbar/Navbar.js

```diff
@@ -47,10 +49,18 @@ export const Navbar = () => {
         </LeftContainer>
         <MiddleContainer>
           <MenuContainer>
-            <NavbarLink to="/Home">Home</NavbarLink>
-            <NavbarLink to="/About">About</NavbarLink>
-            <NavbarLink to="/Contact">Contact</NavbarLink>
-            <NavbarLink to="/FAQ">FAQ</NavbarLink>
+            <NavbarLink data-cy="home" to="/">
+              Home
+            </NavbarLink>
+            <NavbarLink data-cy="about" to="/About">
+              About
+            </NavbarLink>
+            <NavbarLink data-cy="contact" to="/Contact">
+              Contact
+            </NavbarLink>
+            <NavbarLink data-cy="faq" to="/FAQ">
```

Comment:
The route path is `/Faq` in App.js which is different to the Link to `/FAQ`. Should these be the same?